src/modules/dashboard/components/ColonyHome/ColonyExtensions/ColonyExtensions.css

```diff
@@ -24,3 +24,11 @@
 .invisibleLink:hover {
   color: color-mod(var(--dark) alpha(50%));
 }
+
+.loadingText {
+  display: inline-block;
+  margin: -8px 0 0 5px;
```

Comment:
why -8px here? I think it looks better without these margins. The text is way to close to the spinner. Or is it the specs?

Reply:
No. Armando is actually right here.

The `-8px` correctly aligns the text with the spinner